cmd/notation/verify.go

```diff
@@ -56,7 +56,7 @@ Example - Verify a signature on an OCI artifact identified by a tag  (Notation w
 	}
 	opts.LoggingFlagOpts.ApplyFlags(command.Flags())
 	opts.SecureFlagOpts.ApplyFlags(command.Flags())
-	command.Flags().StringArrayVarP(&opts.pluginConfig, "plugin-config", "c", nil, "{key}={value} pairs that are passed as it is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values")
+	command.Flags().StringArrayVar(&opts.pluginConfig, "plugin-config", nil, "{key}={value} pairs that are passed as it is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values")
```

Comment:
why the shorthand "c" was removed?

Reply:
This was a suggestion by @priteshbandi in this [comment](https://github.com/notaryproject/notation/pull/509#discussion_r1081951504).

Reply:
If we have multiple plugin config key=value pairs, a shorthand is helpful. 
```
notation sign xxxxx \
-c key1=value1
-c key2=value2
```
@priteshbandi is this a rational use case that we need the shorthand for plugin-config flag?

Reply:
Update: I think "--plugin-config" is fine.